pyart/graph/gridmapdisplay.py

```diff
@@ -252,6 +248,16 @@ def plot_grid(self, field, level=0, vmin=None, vmax=None,
             ax.add_feature(coastlines, linestyle='-', edgecolor='k',
                            linewidth=2)
 
+        if add_lines:
```

Comment:
Could we name this `add_grid_lines` to make it a bit more clear what sort of lines are being added?